package.json

```diff
@@ -2,7 +2,8 @@
   "name": "bitmovin-javascript",
   "version": "1.2.2",
   "scripts": {
-    "lint": "./node_modules/eslint/bin/eslint.js --color ./bitmovin",
+    "lint": "node ./node_modules/eslint/bin/eslint.js --color ./bitmovin",
```

Comment:
`./node_modules/.bin/eslint` also exists so you would not have to start it with node :)